notebooks/text-generation/CodeLlama-7B-Compilation.ipynb

```diff
@@ -0,0 +1,294 @@
+    "1. Compiling a model\n",
+    "1. Save to disk\n",
+    "\n",
+    "\n",
+    "This Jupyter Notebook can be run on a Inf2 instance (`inf2.xlarge`) or larger or also a Trn1 instance (`trn1.32xlarge`)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a649c112",
+   "metadata": {},
+   "source": [
+    "## Deploy your instance\n",
+    "\n",
+    "1. Raise your limits\n",
+    "    The default limit for Inferentia and Trainium instances are 0, so request an increase before you start!  EC2 instances are limited by vCPUs, but SageMaker limits are based on number of instances and instance size.  Start with a vCPU limit request of 32 so you can deploy an inf2.xlarge (4 vCPUs) or an inf2.8xlarge (32 vCPUs).  Go to [Service Quotas](https://us-west-2.console.aws.amazon.com/servicequotas/home/services/ec2/quotas) and search for `inf`.\n",
+    "\n",
+    "    Request increases for both On-Demand and Spot (yes, there are usually Spot instances available!).  IAD (us-east-1) and PDX (us-west-2) are the best regions to start in for the US.  Limits are region specific!\n",
+    "1. Go to EC2 to launch an instance.  Make sure you choose the region to match your limit increases.\n",
+    "1. Launch an inf2.xlarge instance with the [Hugging Face DLAMI(Deep learning AMI)](https://aws.amazon.com/marketplace/pp/prodview-gr3e6yiscria2).  This image comes with all the software preinstalled.  Alternatively, you can follow the instructions at the [Neuron Setup Guide](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/setup/torch-neuronx.html#setup-torch-neuronx)\n",
+    "1. 100GB disk size is fine for this example, but larger models such as 70B make take over 500GB"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af2b7693-2950-41fc-a038-17cba44bf003",
+   "metadata": {},
+   "source": [
+    "## Set up the Jupyter Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c47ef383-0dea-4423-8c38-29c73927fd78",
+   "metadata": {},
+   "source": [
+    "(You could also just copy and paste most of this code into an SSH prompt)\n",
+    "\n",
+    "The following steps set up Jupyter Notebook and launch this tutorial:\n",
+    "1. Clone the [AWS Neuron Samples](https://github.com/aws-neuron/aws-neuron-samples) repo to your instance using\n",
+    "```\n",
+    "git clone https://github.com/aws-neuron/aws-neuron-samples.git\n",
+    "```\n",
+    "2. Navigate to the `transformers-neuronx` inference samples folder\n",
+    "```\n",
+    "cd aws-neuron-samples/torch-neuronx/transformers-neuronx/inference\n",
+    "```\n",
+    "3. Follow the instructions in [Jupyter Notebook QuickStart](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/setup/notebook/setup-jupyter-notebook-steps-troubleshooting.html) to run Jupyter Notebook on your instance.\n",
+    "4. Locate this tutorial in your Jupyter Notebook session (`CodeLlama-7B.ipynb`) and launch it. Follow the rest of the instructions in this tutorial. "
```

Comment:
I believe this should be fetch from optimum-neuron instead, isn't it ?

Reply:
Yes, I'll fix it.

Reply:
@dacorvo I fixed a number of path issues in that section.  Thanks for the fresh eyes.